src/__test__/components/data-management/FileUploadModal.test.jsx

```diff
@@ -2,19 +2,65 @@ import React from 'react';
 import { Button, Select } from 'antd';
 import { configure, mount, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 import Dropzone from 'react-dropzone';
 import FileUploadModal from '../../../components/data-management/FileUploadModal';
 
 configure({ adapter: new Adapter() });
 
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({});
+
 describe('FileUploadModal', () => {
+  beforeEach(() => {
+    Object.defineProperty(window, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation((query) => ({
+        matches: false,
+        media: query,
+        onchange: null,
+        addListener: jest.fn(), // deprecated
+        removeListener: jest.fn(), // deprecated
+        addEventListener: jest.fn(),
+        removeEventListener: jest.fn(),
+        dispatchEvent: jest.fn(),
+      })),
+    });
+  });
```

Comment:
I don't think you need this anymore - the mock is now loaded by default. Try to see the changes in https://github.com/biomage-ltd/ui/commit/9c813f2d3d586ad3a77cead6aaf385c5ed460715 and test again. You might need to merge the latest branch,